src/components/Stepper/Stepper.tsx

```diff
@@ -23,6 +24,29 @@ export type StepProps = {
   state: StepState
 }
 
+export const getStepState = (stepPage: number, currentStep: number) => {
+  if (stepPage === currentStep) {
+    return "active" as const
+  }
+
+  return currentStep > stepPage ? ("done" as const) : ("todo" as const)
+}
+
+export const LoadgingPage = ({ title }: { title: string }) => (
```

Comment:
typo "Loadging"

Can you move it to separate component? It's not closely related to Stepper.